### ISSUE TITLE
Scroll Mode Document not working in IE11 (Windows 7)

### ISSUE THREAD

**Issue by becka11y** [readium/readium-js-viewer#174]
Tested on Windows 7 using Accessible EPUB 3 book.  In document scroll mode, when using the next page arrow to move through a scrolled spine item, you do not go to the next spine item but are left with a blank, non-working page.

Steps to reproduce in IE 11
1. Open Settings dialog and select Scroll Mode document
2. Open Accessible EPUB 3 book
3. Navigate to chapter 1
4. Press/click the next arrow navigation to scroll the page downward. When you reach then end of the chapter the next chapter should be loaded. It is not. Instead you are left in a blank page that is not functional. 

V1+ since it works in Chrome Extension (as well as other browsers tested)
